### PR TITLE
final Navigation

### PR DESCRIPTION
**Pull request by justincaze**

### CODE REVIEW

File: components/ActivityCard/index.js [hunk 1]
```diff
@@ -21,6 +21,32 @@ export default function ActivityCard({ name, date, time, joined, category }) {
     }
   }, [name]);
 
+  // check date to display date, weekday, "today" or "tomorrow" //////////////////////////////
```
Comment (alex96dev) [inline components/ActivityCard/index.js:24]: I like the idea of commenting functions etc. ^^

File: pages/index.js [hunk 2]
```diff
@@ -170,9 +170,9 @@ export default function HomePage({ onSubmit }) {
               color={theme.textColor}
             />
           </StyledFilterButton>
-          <StyledFriendlistLink href="/friendlist">
+          {/* <StyledFriendlistLink href="/friendlist">
```
Comment (alex96dev) [inline pages/index.js:173]: I guess this is for the case of a rollback?

File: pages/index.js [hunk 3]
```diff
@@ -300,22 +300,22 @@ const StyledFiLogOut = styled(FiLogOut)`
...
-  border-width: ${theme.borderWidth.medium};
-  border-style: solid;
-  box-shadow: ${theme.box.shadowSmall};
-`;
-
-const StyledFiUser = styled(FiUser)`
-  width: 100%;
-  height: 100%;
-`;
+// const StyledFriendlistLink = styled(Link)`
```
Comment (alex96dev) [inline pages/index.js:303]: I guess this is for the case of a rollback?

File: pages/index.js [hunk 4]
```diff
@@ -300,22 +300,22 @@ const StyledFiLogOut = styled(FiLogOut)`
...
+//   border-radius: ${theme.borderRadius.medium};
+//   border-width: ${theme.borderWidth.medium};
+//   border-style: solid;
+//   box-shadow: ${theme.box.shadowSmall};
+// `;
+
+// const StyledFiUser = styled(FiUser)`
+//   width: 100%;
+//   height: 100%;
+// `;
```
Comment (alex96dev) [inline pages/index.js:318]: I guess this is for the case of a rollback?

File: components/ActivityCard/index.js [hunk 5]
```diff
@@ -21,6 +21,32 @@ export default function ActivityCard({ name, date, time, joined, category }) {
...
+      return eventDate.toLocaleDateString("en-US");
+    } else if (eventDate < tomorrow) {
+      return "Tomorrow";
+    } else if (eventDate > oneWeekFromNow) {
+      return eventDate.toLocaleDateString("en-US");
+    } else {
+      return eventDate.toLocaleDateString("en-US", { weekday: "long" });
+    }
+  };
+
```
Comment (alex96dev) [inline components/ActivityCard/index.js:49]: Im wondering why these changes are in this PR too...

**Comment by alex96dev** [comment]
lgtm

anyways I left some comments..

File: components/ActivityCard/index.js (hunk 5 above)
Comment (justincaze) [reply components/ActivityCard/index.js:49]: easy to delete and pure lazyness

File: pages/index.js (hunk 4 above)
Comment (justincaze) [reply pages/index.js:318]: It shouldn't be there. I deleted this block and it's not even in my code anymore

File: components/ActivityCard/index.js (hunk 1 above)
Comment (justincaze) [reply components/ActivityCard/index.js:24]: same
